README.md

```diff
@@ -17,13 +18,40 @@
 * Operates on top of Selenium.
 * The Git for your GUI.
 
+
+### Easy creation and maintenance
```

Comment:
Hmmm, "Easy creation and maintenance" is already above. Any other ideas for a good subsection name? Something like "Advantages" or "Say Goodbye to Assertions"?